src/main/java/Dice.java

```diff
@@ -0,0 +1,20 @@
+import java.util.*;
+
+public class Dice {
+
+    private Random random;
+
+    public Dice(Random random) {
+        this.random = random;
+    }
+
+    public int roll_dice() {
+        return random.nextInt(6);
+    }
+
+     public static void main(String []args) {
+        System.out.println("Rolling a Die");
+        Dice dice = new Dice(new Random());
+        System.out.println("Got a " + dice.roll_dice());
```

Comment:
Can we add an input that states the number of dice to be rolled? This function assumes only one die will always be rolled.

Reply:
Sure - Should there be an upper limit to the number of dice to be rolled?

Reply:
I don't think an upper limit is needed at this time, but we can add one later if we find we need one.

Reply:
I agree with @blondiebytes

Reply:
I agree